src/templates/index.html

```diff
@@ -344,6 +356,230 @@ <h2 class="info-title">Paper Info:</h2>
+                href="https://github.com/matteovisotto/PDFL-PDFLegacy">PDFL-PDFLegacy</a> <br>
+            Checkout also our Chrome plug-in at: <a
+                href="https://chrome.google.com/webstore/detail/pdf-legacy/cnfmfaohminlbglpkfhbofiolmndonkj">PDFL-PDFLegacy
+                Chrome Web Store</a>
+            </p>
+            <h2 id="explore-title">Explore your paper</h2>
+            <p>
+                Read the paper with an extra gear. After you upload a paper you can:
+            <ul>
+                <li>
+                    Navigate through pages
+                    <ul>
+                        <li>
+                            Click on the buttons <i class="material-icons">chevron_left</i><i
+                                class="material-icons">chevron_right</i>
+                        </li>
+                        <li>
+                            Scroll with the mouse
+                        </li>
+                        <li>
+                            Press on the arrow keys on the keyboard
+                        </li>
+                        <li>
+                            Or change page number from the input box
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Zoom in and out
+                    <ul>
+                        <li>
+                            Click on the buttons <i class="material-icons">zoom_in</i> <i
+                                class="material-icons">zoom_out</i>
+                        </li>
+                        <li>
+                            Press ctrl/cmd+ '+' or ctrl/cmd+ '-'
+                        </li>
+                        <li>
+                            Or change percentage from the input box
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Select text from the paper
+                </li>
+                <li>
+                    Reach page faster by clicking on the button &nbsp;<label>&#9776;</label>&nbsp;
+                    and scroll through thumbnails of the pages
+                </li>
+            </ul>
+            </p>
+            <h2 id="cross-references-title">Cross References</h2>
+            <p> Hover the references and extract data from them.
+            <ul>
+                <li>
+                    Finding a reference and hover it
+                    <ul>
+                        <li>
+                            If the reference is about an image you will see it in a small popup and a button
+                        </li>
+                        <li>
+                            If the reference is about a text you will see an extraction of it and a button
+                        </li>
+                        <li>
+                            If the reference is about a page you will see just the button
+                        </li>
```

Comment:
This sentences could be written better, especially since theyre in annotation form.
Maybe something like:
Image reference - a small popup and a button will be displayed
Text reference - _you will see an extraction of it and a button_ -> i dont understand what youre trying to say here exactly, whats an "extraction of text reference"? Maybe this could be worded in a better way...
Page reference - a button will be displayed

Reply:
fixed